android/src/release/kotlin/com/mattermost/networkclient/NetworkClient.kt

```diff
@@ -0,0 +1,22 @@
+package com.mattermost.networkclient
+
+import com.facebook.react.bridge.*
+import com.mattermost.networkclient.interceptors.*
+import okhttp3.*
+import java.net.URI
+
+internal class NetworkClient(baseUrl: HttpUrl? = null, options: ReadableMap? = null, cookieJar: CookieJar? = null): NetworkClientBase(baseUrl) {
+    constructor(webSocketUri: URI, baseUrl: HttpUrl, options: ReadableMap? = null) : this(baseUrl, options) {
```

Comment:
what does that part mean? 
constructor(webSocketUri: URI, baseUrl: HttpUrl, options: ReadableMap? = null) **: this(baseUrl, options)** {

Reply:
Do read about kotlin secondary constructor here https://www.programiz.com/kotlin-programming/constructors